File: 01_basics/01_04_reverse_numbers/Reverse.java

public class Reverse {
    public static void main(String[] args) {
        int n = 23456;
        System.out.println("Reverse of number " + n + " is " + reverse(n));
    }

    private static int reverse(int n) {
        int num = n;
        int reverse = 0;

        while (num > 0) {
            reverse = reverse * 10 + (num % 10);
            num /= 10;
        }

        return reverse;
    }
}

File: 01_basics/01_04_reverse_numbers/reverse.js
const reverseNum = (n) => {
    let x = n;
    let reverse = 0

    while (x > 0) {
        reverse = reverse * 10 + (x % 10);
        x = Math.floor(x / 10);
    }
    return reverse;
}

const num = 56789

console.log("Revers of " + num + " is " + reverseNum(num));